packages/next/src/server/dev/hot-reloader-webpack.ts

```diff
@@ -211,15 +203,21 @@ export async function getVersionInfo(enabled: boolean): Promise<VersionInfo> {
     installed = require('next/package.json').version
 
     const registry = getRegistry()
-    const res = await fetch(`${registry}-/package/next/dist-tags`)
+    let res
+
+    try {
+      res = await fetch(`${registry}-/package/next/dist-tags`)
+    } catch {
+      // ignore fetch errors
```

Comment:
I don't have enough context why the previously enumerated errors were fine and if that equally applies to **all** fetch errors.

Seems to me we should just add the errors that were previously enumerated with their corresponding error code in undici?

Reply:
But there're could be a lot of errors like the ones from https://github.com/nodejs/undici/blob/main/docs/docs/api/Errors.md, if we're seeing another one or undici adding new errors then we need to keep increase the list